Tests/SentryTests/Transaction/SentryTracerTests.swift

```diff
@@ -1180,20 +1180,21 @@ class SentryTracerTests: XCTestCase {
         SentrySDKInternal.setAppStartMeasurement(fixture.getAppStartMeasurement(type: .warm))
         
         let queue = DispatchQueue(label: "", qos: .background, attributes: [.concurrent, .initiallyInactive] )
-        let group = DispatchGroup()
-        
+
         let transactions = 5
+        let expectation = XCTestExpectation(description: "Finish transactions")
+        expectation.expectedFulfillmentCount = transactions
+
         for _ in 0..<transactions {
-            group.enter()
             queue.async {
                 self.fixture.getSut().finish()
-                group.leave()
+                expectation.fulfill()
             }
         }
         
         queue.activate()
-        group.wait()
-        
+        wait(for: [expectation], timeout: 5.0)
```

Comment:
Looks like it should have a higher value since it failed in CI with 5 seconds.
We could use a higher priority queue too since that is not what is being tested but helps making the test more stable.